docs/modules/ROOT/pages/tutorials/migration.adoc

```diff
@@ -0,0 +1,645 @@
+
+. Lowering your TTLs (Time To Live) 24 hours ahead of time will reduce the amount of time that your change takes to propagate. You can limit downtime by lowering that value, so the caching servers will check back more frequently. Typically, 300 seconds (5 minutes) instead of the default of 14400 (4 hours) is a good idea.
++
+[source,bash]
+--
+$ vi /var/named/example.com.db # <1>
+$ rndc reload example.com # <2>
+$ dig @localhost example.com # <3>
+--
+<1> Edit the zone file
+<2> Make BIND aware of your changes
+<3> Test your changes using https://www.liquidweb.com/kb/how-to-use-dig/[dig]
++
+IMPORTANT: Remember to set your TTL back to the previous value after the migration.
+
+. There are different `CNAME` to point to depending on your current zone; please refer to xref:zones.adoc[the APPUiO Cloud Zones] reference document, where the corresponding `CNAME` entry is specified.
+
+== Let's Encrypt
+
+In APPUiO Cloud only `Ingress` objects can request certificates.
+
+IMPORTANT: It's not possible to request certificates in `Route` objects.
+
+The manifest below specifies an `Ingress` object with such requirement:
+
+[source,yaml]
+--
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  annotations:
+    cert-manager.io/cluster-issuer: letsencrypt-production
+    kubernetes.io/ingress.class: nginx
+    kubernetes.io/tls-acme: "true" # <1>
+  name: ${DEPLOYMENT_NAME}
+  namespace: oauth-test
+spec:
+  rules:
+  - host: ${ROUTE_HOST}
+    http:
+      paths:
+      - backend:
+          service:
+            name: nginx
+            port:
+              number: 80
+        path: /
+        pathType: ImplementationSpecific
+  tls:
+  - hosts:
+    - ${ROUTE_HOST}
+    secretName: ${ROUTE_HOST}
+--
+<1> Annotation providing automatic provisioning of certificates on APPUiO Cloud.
+
+For reference, it was possible to configure TLS certificates in `Route` objects in APPUiO Public, using a manifest similar to the following. This is no longer valid in APPUiO Cloud.
+
+[source,yaml]
+--
+- apiVersion: route.openshift.io/v1
+  kind: Route
+  metadata:
+    name: ${DEPLOYMENT_NAME}
+    labels:
+      app: ${CI_ENVIRONMENT_SLUG}
+    annotations:
+      kubernetes.io/tls-acme: "${ROUTE_TLS_ACME}" # <1>
+  spec:
+    host: ${ROUTE_HOST}
+    port:
+      targetPort: http
+    tls:
+      insecureEdgeTerminationPolicy: Redirect
+      termination: edge
+    to:
+        kind: Service
+        name: ${DEPLOYMENT_NAME}
+        weight: 100
+    wildcardPolicy: None
+--
+<1> Annotation providing automatic provisioning of certificates on APPUiO Public.
+
+== Storage
+
+APPUiO Cloud has different requirements regarding storage. This section provides more information.
+
+=== Access the RWO volumes from different pods
+
+On APPUiO Public the default was RWX storage. At the moment, *APPUiO Cloud only features RWO storage*. Former deployments using RWX volumes can't run anymore without modification.
```

Comment:
As we'll be enabling RWX storage in the next two weeks, I'd rather not mention that here as it will be outdated very soon (and the "Ready for Migration" milestone - which this documentation targets - will be in a month).

Reply:
Removed the reference